lib/pure/collections/lists.nim

```diff
@@ -231,7 +224,7 @@ iterator items*[T](L: SomeLinkedRing[T]): T =
     from std/sugar import collect
     from std/sequtils import toSeq
     let a = collect(initSinglyLinkedRing):
-      for i in 1..3: 10*i
+      for i in 1 .. 3: 10 * i
```

Comment:
@xflywind that change is controversial
```nim
1..3
=>
1 .. 3
```

in the past people have done the opposite change; we should maybe not change such instances until there's a guideline for it in `nep1.rst` or `contributing.rst` as a reference guide

Reply:
I would like
-5 .. -1

Reply:
the styleguide rule could be:
use `a..b` except if `b` involves an operator (eg -1)

Reply:
=> https://github.com/nim-lang/Nim/pull/16992